src/main/java/org/example/examplefinalProject/entity/Parent.java

```diff
@@ -20,14 +21,13 @@ public class Parent {
     @ManyToMany(fetch = FetchType.EAGER, cascade = {CascadeType.DETACH, CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH})
     @JoinTable(
             name = "student_parent",
-            joinColumns = @JoinColumn(name = "parentId"),
-            inverseJoinColumns = @JoinColumn(name = "studentId")
```

Comment:
studentId jsi změnil na student_id trochu tomu nerozumím

Reply:
v mysql je to pod parent_id zaroven aj vo flyway scripte